### PR TITLE
[Style] Fix line lengths in action.admin.indices

### PR DESCRIPTION
**Pull request by AthenaEryma**
Clean up lines over 140 characters in the the
`org.elasticsearch.action.admin.indices` packages

Relates to #34884

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/action/admin/indices/cache/clear/ClearIndicesCacheRequestBuilder.java [hunk 1]
```diff
@@ -22,7 +22,8 @@
 import org.elasticsearch.action.support.broadcast.BroadcastOperationRequestBuilder;
 import org.elasticsearch.client.ElasticsearchClient;
 
-public class ClearIndicesCacheRequestBuilder extends BroadcastOperationRequestBuilder<ClearIndicesCacheRequest, ClearIndicesCacheResponse, ClearIndicesCacheRequestBuilder> {
+public class ClearIndicesCacheRequestBuilder
+    extends BroadcastOperationRequestBuilder<ClearIndicesCacheRequest, ClearIndicesCacheResponse, ClearIndicesCacheRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/cache/clear/ClearIndicesCacheRequestBuilder.java]: Could you indent this one more time so it doesn't line up with the class body?

File: server/src/main/java/org/elasticsearch/action/admin/indices/delete/DeleteIndexRequestBuilder.java [hunk 2]
```diff
@@ -24,7 +24,8 @@
 import org.elasticsearch.action.support.master.AcknowledgedResponse;
 import org.elasticsearch.client.ElasticsearchClient;
 
-public class DeleteIndexRequestBuilder extends AcknowledgedRequestBuilder<DeleteIndexRequest, AcknowledgedResponse, DeleteIndexRequestBuilder> {
+public class DeleteIndexRequestBuilder
+    extends AcknowledgedRequestBuilder<DeleteIndexRequest, AcknowledgedResponse, DeleteIndexRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/delete/DeleteIndexRequestBuilder.java]: Here too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/exists/indices/IndicesExistsRequestBuilder.java [hunk 3]
```diff
@@ -22,7 +22,8 @@
 import org.elasticsearch.action.support.master.MasterNodeReadOperationRequestBuilder;
 import org.elasticsearch.client.ElasticsearchClient;
 
-public class IndicesExistsRequestBuilder extends MasterNodeReadOperationRequestBuilder<IndicesExistsRequest, IndicesExistsResponse, IndicesExistsRequestBuilder> {
+public class IndicesExistsRequestBuilder
+    extends MasterNodeReadOperationRequestBuilder<IndicesExistsRequest, IndicesExistsResponse, IndicesExistsRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/exists/indices/IndicesExistsRequestBuilder.java]: Here too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/exists/types/TypesExistsRequestBuilder.java [hunk 4]
```diff
@@ -27,7 +27,8 @@
  * A builder for {@link TypesExistsRequest}.
  */
 @Deprecated
-public class TypesExistsRequestBuilder extends MasterNodeReadOperationRequestBuilder<TypesExistsRequest, TypesExistsResponse, TypesExistsRequestBuilder> {
+public class TypesExistsRequestBuilder
+    extends MasterNodeReadOperationRequestBuilder<TypesExistsRequest, TypesExistsResponse, TypesExistsRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/exists/types/TypesExistsRequestBuilder.java]: Here too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/flush/TransportFlushAction.java [hunk 5]
```diff
@@ -35,13 +35,15 @@
 /**
  * Flush Action.
  */
-public class TransportFlushAction extends TransportBroadcastReplicationAction<FlushRequest, FlushResponse, ShardFlushRequest, ReplicationResponse> {
+public class TransportFlushAction
+    extends TransportBroadcastReplicationAction<FlushRequest, FlushResponse, ShardFlushRequest, ReplicationResponse> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/flush/TransportFlushAction.java]: And here too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/flush/TransportShardFlushAction.java [hunk 6]
```diff
@@ -32,7 +32,8 @@
 import org.elasticsearch.threadpool.ThreadPool;
 import org.elasticsearch.transport.TransportService;
 
-public class TransportShardFlushAction extends TransportReplicationAction<ShardFlushRequest, ShardFlushRequest, ReplicationResponse> {
+public class TransportShardFlushAction
+    extends TransportReplicationAction<ShardFlushRequest, ShardFlushRequest, ReplicationResponse> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/flush/TransportShardFlushAction.java]: And here too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/forcemerge/ForceMergeRequestBuilder.java [hunk 7]
```diff
@@ -29,7 +29,8 @@
  * merge down to. By default, will cause the force merge process to merge down
  * to half the configured number of segments.
  */
-public class ForceMergeRequestBuilder extends BroadcastOperationRequestBuilder<ForceMergeRequest, ForceMergeResponse, ForceMergeRequestBuilder> {
+public class ForceMergeRequestBuilder
+    extends BroadcastOperationRequestBuilder<ForceMergeRequest, ForceMergeResponse, ForceMergeRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/forcemerge/ForceMergeRequestBuilder.java]: And here too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/forcemerge/TransportForceMergeAction.java [hunk 8]
```diff
@@ -43,7 +43,8 @@
 /**
  * ForceMerge index/indices action.
  */
-public class TransportForceMergeAction extends TransportBroadcastByNodeAction<ForceMergeRequest, ForceMergeResponse, TransportBroadcastByNodeAction.EmptyResult> {
+public class TransportForceMergeAction
+    extends TransportBroadcastByNodeAction<ForceMergeRequest, ForceMergeResponse, TransportBroadcastByNodeAction.EmptyResult> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/forcemerge/TransportForceMergeAction.java]: And another one!

File: server/src/main/java/org/elasticsearch/action/admin/indices/mapping/get/GetFieldMappingsRequestBuilder.java [hunk 9]
```diff
@@ -25,7 +25,8 @@
 import org.elasticsearch.common.util.ArrayUtils;
 
 /** A helper class to build {@link GetFieldMappingsRequest} objects */
-public class GetFieldMappingsRequestBuilder extends ActionRequestBuilder<GetFieldMappingsRequest, GetFieldMappingsResponse> {
+public class GetFieldMappingsRequestBuilder
+    extends ActionRequestBuilder<GetFieldMappingsRequest, GetFieldMappingsResponse> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/mapping/get/GetFieldMappingsRequestBuilder.java]: This one too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/mapping/get/GetMappingsRequestBuilder.java [hunk 10]
```diff
@@ -22,7 +22,8 @@
 import org.elasticsearch.action.support.master.info.ClusterInfoRequestBuilder;
 import org.elasticsearch.client.ElasticsearchClient;
 
-public class GetMappingsRequestBuilder extends ClusterInfoRequestBuilder<GetMappingsRequest, GetMappingsResponse, GetMappingsRequestBuilder> {
+public class GetMappingsRequestBuilder
+    extends ClusterInfoRequestBuilder<GetMappingsRequest, GetMappingsResponse, GetMappingsRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/mapping/get/GetMappingsRequestBuilder.java]: And this one too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/mapping/get/TransportGetFieldMappingsIndexAction.java [hunk 11]
```diff
@@ -58,7 +58,8 @@
 /**
  * Transport action used to retrieve the mappings related to fields that belong to a specific index
  */
-public class TransportGetFieldMappingsIndexAction extends TransportSingleShardAction<GetFieldMappingsIndexRequest, GetFieldMappingsResponse> {
+public class TransportGetFieldMappingsIndexAction
+    extends TransportSingleShardAction<GetFieldMappingsIndexRequest, GetFieldMappingsResponse> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/mapping/get/TransportGetFieldMappingsIndexAction.java]: And this one!

File: server/src/main/java/org/elasticsearch/action/admin/indices/segments/TransportIndicesSegmentsAction.java [hunk 12]
```diff
@@ -41,13 +41,15 @@
 import java.io.IOException;
 import java.util.List;
 
-public class TransportIndicesSegmentsAction extends TransportBroadcastByNodeAction<IndicesSegmentsRequest, IndicesSegmentResponse, ShardSegments> {
+public class TransportIndicesSegmentsAction
+    extends TransportBroadcastByNodeAction<IndicesSegmentsRequest, IndicesSegmentResponse, ShardSegments> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/segments/TransportIndicesSegmentsAction.java]: This one too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/segments/IndicesSegmentsRequestBuilder.java [hunk 13]
```diff
@@ -22,7 +22,8 @@
 import org.elasticsearch.action.support.broadcast.BroadcastOperationRequestBuilder;
 import org.elasticsearch.client.ElasticsearchClient;
 
-public class IndicesSegmentsRequestBuilder extends BroadcastOperationRequestBuilder<IndicesSegmentsRequest, IndicesSegmentResponse, IndicesSegmentsRequestBuilder> {
+public class IndicesSegmentsRequestBuilder
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/segments/IndicesSegmentsRequestBuilder.java]: And this one!

File: server/src/main/java/org/elasticsearch/action/admin/indices/settings/get/GetSettingsRequestBuilder.java [hunk 14]
```diff
@@ -24,7 +24,8 @@
 import org.elasticsearch.client.ElasticsearchClient;
 import org.elasticsearch.common.util.ArrayUtils;
 
-public class GetSettingsRequestBuilder extends MasterNodeReadOperationRequestBuilder<GetSettingsRequest, GetSettingsResponse, GetSettingsRequestBuilder> {
+public class GetSettingsRequestBuilder
+    extends MasterNodeReadOperationRequestBuilder<GetSettingsRequest, GetSettingsResponse, GetSettingsRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/settings/get/GetSettingsRequestBuilder.java]: This one too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/settings/put/UpdateSettingsRequestBuilder.java [hunk 15]
```diff
@@ -31,7 +31,8 @@
 /**
  * Builder for an update index settings request
  */
-public class UpdateSettingsRequestBuilder extends AcknowledgedRequestBuilder<UpdateSettingsRequest, AcknowledgedResponse, UpdateSettingsRequestBuilder> {
+public class UpdateSettingsRequestBuilder
+    extends AcknowledgedRequestBuilder<UpdateSettingsRequest, AcknowledgedResponse, UpdateSettingsRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/settings/put/UpdateSettingsRequestBuilder.java]: This one too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/shards/IndicesShardStoreRequestBuilder.java [hunk 16]
```diff
@@ -28,7 +28,8 @@
 /**
  * Request builder for {@link IndicesShardStoresRequest}
  */
-public class IndicesShardStoreRequestBuilder extends MasterNodeReadOperationRequestBuilder<IndicesShardStoresRequest, IndicesShardStoresResponse, IndicesShardStoreRequestBuilder> {
+public class IndicesShardStoreRequestBuilder
+    extends MasterNodeReadOperationRequestBuilder<IndicesShardStoresRequest, IndicesShardStoresResponse, IndicesShardStoreRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/shards/IndicesShardStoreRequestBuilder.java]: And another one!

File: server/src/main/java/org/elasticsearch/action/admin/indices/shards/TransportIndicesShardStoresAction.java [hunk 17]
```diff
@@ -59,17 +59,21 @@
...
 
 /**
- * Transport action that reads the cluster state for shards with the requested criteria (see {@link ClusterHealthStatus}) of specific indices
- * and fetches store information from all the nodes using {@link TransportNodesListGatewayStartedShards}
+ * Transport action that reads the cluster state for shards with the requested criteria (see {@link ClusterHealthStatus}) of specific
+ * indices and fetches store information from all the nodes using {@link TransportNodesListGatewayStartedShards}
  */
-public class TransportIndicesShardStoresAction extends TransportMasterNodeReadAction<IndicesShardStoresRequest, IndicesShardStoresResponse> {
+public class TransportIndicesShardStoresAction
+    extends TransportMasterNodeReadAction<IndicesShardStoresRequest, IndicesShardStoresResponse> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/shards/TransportIndicesShardStoresAction.java]: This one too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/stats/IndicesStatsRequestBuilder.java [hunk 18]
```diff
@@ -31,7 +31,8 @@
  * All the stats to be returned can be cleared using {@link #clear()}, at which point, specific
  * stats can be enabled.
  */
-public class IndicesStatsRequestBuilder extends BroadcastOperationRequestBuilder<IndicesStatsRequest, IndicesStatsResponse, IndicesStatsRequestBuilder> {
+public class IndicesStatsRequestBuilder
+    extends BroadcastOperationRequestBuilder<IndicesStatsRequest, IndicesStatsResponse, IndicesStatsRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/stats/IndicesStatsRequestBuilder.java]: And this one!

File: server/src/main/java/org/elasticsearch/action/admin/indices/template/delete/DeleteIndexTemplateRequestBuilder.java [hunk 19]
```diff
@@ -22,7 +22,8 @@
 import org.elasticsearch.action.support.master.MasterNodeOperationRequestBuilder;
 import org.elasticsearch.client.ElasticsearchClient;
 
-public class DeleteIndexTemplateRequestBuilder extends MasterNodeOperationRequestBuilder<DeleteIndexTemplateRequest, AcknowledgedResponse, DeleteIndexTemplateRequestBuilder> {
+public class DeleteIndexTemplateRequestBuilder
+    extends MasterNodeOperationRequestBuilder<DeleteIndexTemplateRequest, AcknowledgedResponse, DeleteIndexTemplateRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/template/delete/DeleteIndexTemplateRequestBuilder.java]: And this one too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/template/delete/TransportDeleteIndexTemplateAction.java [hunk 20]
```diff
@@ -37,15 +37,17 @@
 /**
  * Delete index action.
  */
-public class TransportDeleteIndexTemplateAction extends TransportMasterNodeAction<DeleteIndexTemplateRequest, AcknowledgedResponse> {
+public class TransportDeleteIndexTemplateAction
+    extends TransportMasterNodeAction<DeleteIndexTemplateRequest, AcknowledgedResponse> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/template/delete/TransportDeleteIndexTemplateAction.java]: And this one too!

File: server/src/main/java/org/elasticsearch/action/admin/indices/template/get/GetIndexTemplatesRequestBuilder.java [hunk 21]
```diff
@@ -21,7 +21,8 @@
 import org.elasticsearch.action.support.master.MasterNodeReadOperationRequestBuilder;
 import org.elasticsearch.client.ElasticsearchClient;
 
-public class GetIndexTemplatesRequestBuilder extends MasterNodeReadOperationRequestBuilder<GetIndexTemplatesRequest, GetIndexTemplatesResponse, GetIndexTemplatesRequestBuilder> {
+public class GetIndexTemplatesRequestBuilder
+    extends MasterNodeReadOperationRequestBuilder<GetIndexTemplatesRequest, GetIndexTemplatesResponse, GetIndexTemplatesRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/template/get/GetIndexTemplatesRequestBuilder.java]: And this one!

File: server/src/main/java/org/elasticsearch/action/admin/indices/upgrade/get/UpgradeStatusRequestBuilder.java [hunk 22]
```diff
@@ -22,7 +22,8 @@
 import org.elasticsearch.action.support.broadcast.BroadcastOperationRequestBuilder;
 import org.elasticsearch.client.ElasticsearchClient;
 
-public class UpgradeStatusRequestBuilder extends BroadcastOperationRequestBuilder<UpgradeStatusRequest, UpgradeStatusResponse, UpgradeStatusRequestBuilder> {
+public class UpgradeStatusRequestBuilder
+    extends BroadcastOperationRequestBuilder<UpgradeStatusRequest, UpgradeStatusResponse, UpgradeStatusRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/upgrade/get/UpgradeStatusRequestBuilder.java]: And this one!

File: server/src/main/java/org/elasticsearch/action/admin/indices/upgrade/post/UpgradeSettingsRequestBuilder.java [hunk 23]
```diff
@@ -30,7 +30,8 @@
 /**
  * Builder for an update index settings request
  */
-public class UpgradeSettingsRequestBuilder extends AcknowledgedRequestBuilder<UpgradeSettingsRequest, AcknowledgedResponse, UpgradeSettingsRequestBuilder> {
+public class UpgradeSettingsRequestBuilder
+    extends AcknowledgedRequestBuilder<UpgradeSettingsRequest, AcknowledgedResponse, UpgradeSettingsRequestBuilder> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/upgrade/post/UpgradeSettingsRequestBuilder.java]: And this one!

File: server/src/main/java/org/elasticsearch/action/admin/indices/validate/query/TransportValidateQueryAction.java [hunk 24]
```diff
@@ -59,7 +59,8 @@
 import java.util.concurrent.atomic.AtomicReferenceArray;
 import java.util.function.LongSupplier;
 
-public class TransportValidateQueryAction extends TransportBroadcastAction<ValidateQueryRequest, ValidateQueryResponse, ShardValidateQueryRequest, ShardValidateQueryResponse> {
+public class TransportValidateQueryAction
+    extends TransportBroadcastAction<ValidateQueryRequest, ValidateQueryResponse, ShardValidateQueryRequest, ShardValidateQueryResponse> {
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/action/admin/indices/validate/query/TransportValidateQueryAction.java]: And this one!

**Review by nik9000 — APPROVED** [review]
LGTM except for 24 place I'd like you to hit tab one extra time so the class body doesn't mix in with the class declaration.

**Review by dakrone — APPROVED** [review]
LGTM assuming CI is happy :)

**Comment by gwbrown** [comment]
I messed up the commit message on the backport and left out the PR number so it didn't get linked here, the backport of this is e30bee73e093e722a71e51686cf726ac4547f2b8

**Pull request merged**